dali/python/nvidia/dali/types.py

```diff
@@ -55,15 +55,17 @@ def _not_implemented(val):
         DALIDataType.INTERP_TYPE : ("nvidia.dali.types.DALIInterpType", lambda x: DALIInterpType(int(x))),
         DALIDataType.TENSOR_LAYOUT : (":ref:`layout str<layout_str_doc>`", lambda x: str(x)),
         DALIDataType.PYTHON_OBJECT : ("object", lambda x: x),
-        DALIDataType._TENSOR_LAYOUT_VEC : (":ref:`layout str<layout_str_doc>`", _to_list(lambda x: str(x)))
+        DALIDataType._TENSOR_LAYOUT_VEC : (":ref:`layout str<layout_str_doc>`", _to_list(lambda x: str(x))),
+        DALIDataType._DALI_DATA_TYPE_VEC : ("DALIDataType", _to_list(lambda x: str(DALIDataType.DATA_TYPE)))
```

Comment:
I'd say that this line is probably not needed, as it should go `_vector_types` and map _DALI_DATA_TYPE_VEC to a list of DALIDataType.DATA_TYPE

Reply:
If it is indeed necessary, it is wrong, as the encoding uses string and not int as the scalar DALIDataType does. It probably works due to not being used.

Reply:
Other vector types go here as well. In `_vector_types` we are holding type of vector's element.

Reply:
That lambda is not used, but we still need to add this vector here.

Reply:
Right, my mistake.